app/models/concerns/operations/stock_operations.rb

```diff
@@ -123,6 +126,10 @@ def error(error)
         { errors: [error], success: false }
       end
 
+      def box_dispatched?
```

Comment:
As discussed on slack with @abhinav098 , we'll change this to check for `available_quantity > 0` - this is the amount that is left over once a package has been designated or dispatched. The Inventory computer can supply this value direct